great-idea-website/css/index.css

```diff
@@ -63,4 +63,92 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+body{
+	padding:200px;
+}
+
+
+a {
+display:inline-block;
+color:gray;
+font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+text-decoration:none;
+margin:30px 35px;
+}
+
+img {
+	display:inline;
+}
+
+div{
```

Comment:
This div style might override other styles. I'll advise that you give each div their own class to make it easily selectable.